src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,86 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int CAPACITY = 16;
+    private int size;
+    private int threshold;
+    private Node<K, V>[] table;
+
+    private class Node<K, V> {
+        private final K key;
+        private V value;
+        private Node<K, V> next;
+
+        public Node(K key, V value, Node<K, V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
+
+    public MyHashMap() {
+        threshold = (int) (DEFAULT_LOAD_FACTOR * CAPACITY);
+        table = new Node[CAPACITY];
+    }
+
+    public int hash(K key) {
```

Comment:
```suggestion
    public int getIndex(K key) {
```
ми тут не тільки хешуємо ключ